src/services/subgraph/queries.ts

```diff
@@ -1,3 +1,9 @@
+const YVBOOST = "0x9d409a0a012cfba9b15f6d4b36ac57a46966ab9a";
+const YVECRV = "0xc5bddf9843308380375a611c18b50fb9341f502a";
+const PSLPYVBOOSTETH = "0xced67a187b923f0e5ebcc77c7f2f7da20099e378";
```

Comment:
It's a shame we can't get a list of all `special` vaults from the registry, these are still categorized as 'v2' vaults. Think it's ok to hardcode it here like this though as this is just for the earnings

Reply:
Yeah same here.

Should we have a `constants` file that exports this addresses on the sdk?

In that way all the other consumers on the sdk don't have to hardcode these vault addressess too

Reply:
I think that'd be good but we'll need to make sure we don't have too much in there - ideally all data would be provided from on-chain sources and we wouldn't need to hardcode things